### PR TITLE
Clean up unused loads

### PR DESCRIPTION
**Pull request by kastiglione**
This contains no functional changes, only removals of unused `load`ed symbols.

### CODE REVIEW

File: swift/internal/swift_binary_test.bzl [hunk 1]
```diff
@@ -19,7 +19,7 @@ load(":derived_files.bzl", "derived_files")
 load(":features.bzl", "SWIFT_FEATURE_BUNDLED_XCTESTS", "is_feature_enabled")
 load(":linking.bzl", "register_link_action")
 load(":providers.bzl", "SwiftBinaryInfo", "SwiftToolchainInfo")
-load(":utils.bzl", "expand_locations", "get_optionally")
```
Comment (kastiglione) [inline swift/internal/swift_binary_test.bzl]: Note this is the last use of `get_optionally`, and so could be deleted.

**Comment by allevato** [comment]
Thanks! I pulled this internally to test, fixed up the conflicts, and removed `get_optionally`—#70 will close this.

**Comment by kastiglione** [comment]
thanks @allevato 

> fixed up the conflicts

conflicts with yet unpushed internal changes?

**Comment by allevato** [comment]
No; it's not visible anymore in the GitHub UI, but some changes I had pushed in the past few days had introduced merge conflicts with your PR since you had written it.

**Pull request closed without merging**